frontend/src/components/popup/SectionDetails.tsx

```diff
@@ -1,29 +1,56 @@
 import * as APIv4 from "hyperschedule-shared/api/v4";
 import Css from "./SectionDetails.module.css";
 import CourseDescriptionBox from "@components/course-search/CourseDescriptionBox";
-import { useActiveSectionsLookup } from "@hooks/section";
+// import { useActiveSectionsLookup } from "@hooks/section";
 import { useUserStore } from "@hooks/store/user";
 import { memo } from "react";
 
 export default memo(function SectionDetails(props: {
-    section: APIv4.SectionIdentifier;
+    section: APIv4.Section | undefined;
 }): JSX.Element {
     const activeTerm = useUserStore((store) => store.activeTerm);
 
-    const lookup = useActiveSectionsLookup();
-    const s = lookup.get(APIv4.stringifySectionCodeLong(props.section));
-    if (s === undefined) return <>Section data not found 😔</>;
+    if (props.section === undefined) return <>Section data not found 😔</>;
+
+    const fromActiveTerm =
+        activeTerm.term === props.section.identifier.term &&
+        activeTerm.year === props.section.identifier.year;
     return (
         <div className={Css.content}>
             <h3 className={Css.sectionTitle}>
-                {(activeTerm.term === props.section.term &&
-                    activeTerm.year === props.section.year
+                {(fromActiveTerm
                     ? APIv4.stringifySectionCode
-                    : APIv4.stringifySectionCodeLong)(props.section)}
-                : {s.course.title}
+                    : APIv4.stringifySectionCodeLong)(props.section.identifier)}
+                : {props.section.course.title}
             </h3>
 
-            <CourseDescriptionBox section={s} showStatus={true} />
+            <CourseDescriptionBox
+                section={props.section}
+                showStatus={fromActiveTerm}
+            />
         </div>
     );
 });
+
+// export default memo(function SectionDetails(props: {
+//     section: APIv4.SectionIdentifier;
+// }): JSX.Element {
+//     const activeTerm = useUserStore((store) => store.activeTerm);
+
+//     const lookup = useActiveSectionsLookup();
+//     const s = lookup.get(APIv4.stringifySectionCodeLong(props.section));
+//     if (s === undefined) return <>Section data not found 😔</>;
+//     return (
+//         <div className={Css.content}>
+//             <h3 className={Css.sectionTitle}>
+//                 {(activeTerm.term === props.section.term &&
+//                     activeTerm.year === props.section.year
+//                     ? APIv4.stringifySectionCode
+//                     : APIv4.stringifySectionCodeLong)(props.section)}
+//                 : {s.course.title}
+//             </h3>
+
+//             <CourseDescriptionBox section={s} showStatus={true} />
+//         </div>
+//     );
+// });
```

Comment:
don't comment out big blocks of code like this. if we don't need it anymore, delete it. if it turned out we need to revert to an earlier version we can get it from git